keanu-python/tests/test_appveyor_hanging.py

```diff
@@ -0,0 +1,25 @@
+import keanu as kn
+import numpy as np
+import sys
+import logging
+
+
+def test_appveyor_hanging():
```

Comment:
Do we need this now that AppVeyor is no longer hanging - or do you want to keep it just incase it starts happening again?

Reply:
Good call. I will remove it and we can always refer back to this PR if we need to reinstate it.